capstone/target/appengine-staging/WEB-INF/classes/source-context.json

```diff
@@ -0,0 +1,5 @@
+<<<<<<< HEAD
```

Comment:
Please merge in Jerry's change to add target directory from gitignore file and then remove the target directory from this pull request.